include/RE/S/ShadowState.h

```diff
@@ -266,89 +266,97 @@ namespace RE
 				return g_RendererShadowState;
 			}
 
+#define GET_RUNTIME_MEMBER(a_value) \
```

Comment:
I feel like this should be in some other common location. Can happen later though.

Reply:
Yeah, I was thinking of maybe we can move the CS macro to clib as well maybe?